### PR TITLE
Uplift doc pages with admonitions

### PR DESCRIPTION
**Pull request by andabakus**

### CODE REVIEW

**Review by Kahonnohak — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]
Awesome! LGTM

File: docs/dapp-staking/for-devs/requirements.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ sidebar_position: 1
 
 Are you interested to join dApp staking on Shiden or Astar? Read carefully to see if you meet the requirements:
 
-> Astar is aiming to become a DAO in a couple of years. While moving towards becoming a DAO our dApp staking onboarding also includes passing our council. The process is to create your proposal to join dApp staking also on our [forum](https://forum.astar.network/c/builders-program/proposals/16) after meeting the requirements below.
+:::info
+Astar is aiming to become a DAO in a couple of years. While moving towards becoming a DAO our dApp staking onboarding also includes passing our council. The process is to create your proposal to join dApp staking also on our [forum](https://forum.astar.network/c/builders-program/proposals/16) after meeting the requirements below.
```
Comment (fiexer) [inline docs/dapp-staking/for-devs/requirements.md:12]: Correct link to the forum should be:
https://forum.astar.network/c/initiatives/dapp-staking-applications/21

File: docs/integrations/dapp-listing/defillama.md [hunk 2]
```diff
@@ -88,7 +88,9 @@ SDK adapters always export balance objects, which is a dictionary where all the
 
 If a token balance has an address key, the Defi Llama SDK will manage any raw to real amount conversion for you (so you don't need to worry about ERC20 decimals). If a token balance has a Coingecko ID key, you will need to process the decimals and use a real token amount in the balances object.
 
-> If you export token addresses in your balances object that isn't on CoinGecko, Defi Llama won't be able to fetch prices for the tokens. You can check which addresses are supported by going to the token on CoinGecko and checking the  'Contract' field on the right (pictured above).
+:::caution
+If you export token addresses in your balances object that isn't on CoinGecko, Defi Llama won't be able to fetch prices for the tokens. You can check which addresses are supported by going to the token on CoinGecko and checking the  'Contract' field on the right (pictured above).
```
Comment (fiexer) [inline docs/integrations/dapp-listing/defillama.md:92]: Please change into (small text change):
If you export token addresses in your balances object that isn't on CoinGecko, DefiLlama won't be able to fetch prices for the tokens. You can check which addresses are supported by going to the token on CoinGecko and checking the 'Contract' field on the right (pictured above).

File: docs/integrations/dapp-listing/defillama.md [hunk 3]
```diff
@@ -143,7 +145,9 @@ module.exports = {
 
 Fetch adapters only allow us to get the TVL at the current time, so it's impossible to fill old values on a protocol's TVL chart or recompute them, thus leading to charts that look jumpy. To solve this we introduced SDK adapters, which allow us to retrieve a protocol's TVL at any point in time.
 
-> Fetch adapters can only be used for projects on non-EVM chains. Where possible, [SDK adapters](https://app.gitbook.com/o/-LgGrgOEDyFYjYWIb1DT/s/-M8GVK5H7hOsGnYqg-7q-872737601/integration/dapp-listing/defillama) are preferred to fetch adapters because on-chain calls are more transparent.
+:::caution
+Fetch adapters can only be used for projects on non-EVM chains. Where possible, [SDK adapters](https://app.gitbook.com/o/-LgGrgOEDyFYjYWIb1DT/s/-M8GVK5H7hOsGnYqg-7q-872737601/integration/dapp-listing/defillama) are preferred to fetch adapters because on-chain calls are more transparent.
```
Comment (fiexer) [inline docs/integrations/dapp-listing/defillama.md:149]: Please link the SDK adapters to: https://docs.llama.fi/list-your-project/how-to-write-a-fetch-adapter

File: docs/dapp-staking/for-devs/requirements.md (hunk 1 above)
Comment (SUGURUSEO) [reply docs/dapp-staking/for-devs/requirements.md:12]: This change was already made in the main branch so merging should fix the issue.

File: docs/integrations/dapp-listing/defillama.md (hunk 3 above)
Comment (andabakus) [reply docs/integrations/dapp-listing/defillama.md]: updated.

File: docs/integrations/dapp-listing/defillama.md (hunk 2 above)
Comment (andabakus) [reply docs/integrations/dapp-listing/defillama.md]: done

**Pull request merged**